src/components/PatientRecord/PatientRecord.tsx

```diff
@@ -143,4 +150,39 @@ const Visualizer: FC<VisualizerProps> = ({ resourceType, resourcesByType }) => {
   else return <div>Unsupported Resource</div>;
 };
 
+const PathwayVisualizer: FC = () => {
+  const resources = usePatientRecords().patientRecords;
+  const evaluatedPathway = usePathwayContext().evaluatedPathway;
+  const [criteria, setCriteria] = useState<CriteriaResult | null>(null);
+
+  useEffect(() => {
+    // Create a fake Bundle for the CQL engine and check if patientPath needs to be evaluated
```

Comment:
the word `fake` in here confused me for a sec as I was thinking it didn't actually have the patient records in it, but it looks like it does (unless I'm just reading it wrong).
```suggestion
    // Create a Bundle for the CQL engine and check if patientPath needs to be evaluated
```

Reply:
Updated. This comment was copied so it was updated in `Graph.tsx` and `PathwayList.tsx` as well